build.gradle

```diff
@@ -79,8 +88,39 @@ configure(javaProjects) {
     }
 }
 
-configure(javaMainProjects) {
+configure(scalaProjects) {
+    apply plugin: 'scala'
+    apply plugin: 'osgi'
     apply plugin: 'idea'
+    apply plugin: "com.adtran.scala-multiversion-plugin"
+    apply plugin: "com.diffplug.gradle.spotless"
```

Comment:
That is because the spotless changes havent been applied yet.

> Note: ./gradlew spotlessCheck will fail I was planning on applying the spotless changes after this PR.

Reply:
I've now applied the changes in a single commit